src/components/LanguageMenu/LanguageMenu.tsx

```diff
@@ -62,7 +62,10 @@ export interface LanguageMenuProps {
     OptionalLanguageMenuPopoverProps
   >;
   /** Menu items: MenuItem or MenuLink */
-  children?: Array<React.ReactElement<LanguageMenuPopoverItemsProps>>;
+  children?:
```

Comment:
Could this cause unwanted side-effects? This was fixed due to type error after updating TS and linters.

Reply:
As the prop is optional, this change does not affect API behavior. However, leaving out children completely has a side-effect of not rendering the menu at all. I'm wondering if we should make the prop mandatory and remove the replace-component wrapper completely.